samples/bluetooth/nrf_auraconfig/prj.conf

```diff
@@ -98,6 +93,8 @@ CONFIG_SHELL_VT100_COMMANDS=y
 CONFIG_SHELL_VT100_COLORS=y
 CONFIG_SHELL_STACK_SIZE=8096
 CONFIG_SHELL_CMD_BUFF_SIZE=128
+CONFIG_SHELL_DEFAULT_TERMINAL_WIDTH=120
```

Comment:
Discuss: How this relates to CONFIG_FS_FATFS_MAX_LFN

Reply:
This can only be properly resolved by https://github.com/zephyrproject-rtos/zephyr/issues/81665